students/ThanhNhan/README.md

```diff
@@ -0,0 +1,22 @@
+# Class Repository
```

Comment:
did you copy this from the parent Dir?

Anyone, not problem -- you can put anything you want in this file -- it's yours :-)